lib/pages/people-page.js

```diff
@@ -94,19 +94,17 @@ export default class PeoplePage extends BaseContainer {
 	 * @param {bool} useSince Whether to include the "SINCE" line in the pattern match (for Followers, not Viewers)
 	 * @returns {Promise} Promise from the click action
 	 */
-	removeUserByName( username, useSince ) {
+	removeUserByName( username ) {
 		let viewers = this.driver.findElements( By.css( '.people-list-item' ) );
 
 		webdriver.promise.filter( viewers, function( viewer ) {
-			return viewer.getText().then( function( text ) {
-				let re = new RegExp( `${username}\n@${username}\nRemove` );
-
-				if ( useSince ) {
-					re = new RegExp( `${username}\n@${username}\nSINCE.*\nREMOVE` );
-				}
-
-				return text.match( re );
-			} );
+			return viewer.findElement( By.css( '.people-profile__username' ) )
+				.then( ( profileUsernameElement ) => {
```

Comment:
what about to convert it into a one-liner `( profileUsernameElement => profileUsernameElement.getText() )?

Reply:
Thanks! updated